### PR TITLE
Fix/timeout

### PR DESCRIPTION
**Pull request by salanza**
Changes based on issue#8

### CODE REVIEW

**Comment by jzarca01** [comment]
```
constructor(options = {}) {
    this.request = axios.create({
            baseURL: 'https://api.birdapp.com',
            headers: {
                'Device-id': faker.random.uuid(),
                'Platform': 'ios',
                'App-Version': '3.0.5'
            },
            ...options
    })
}
```

Options can then be
```
{
    timeout: 0,
    headers: {
        'User-Agent': 'sdfsddddssfs',
        'App-Version': 'this new app version i didnt plan when i first built this api'
    }
    ...whatever else user needs
}
```

Plus i dont care about your documentation if it's only for constructor

**Comment by salanza** [comment]
> ```
> constructor(options = {}) {
>     this.request = axios.create({
>             baseURL: 'https://api.birdapp.com',
>             headers: {
>                 'Device-id': faker.random.uuid(),
>                 'Platform': 'ios',
>                 'App-Version': '3.0.5'
>             },
>             ...options
>     })
> }
> ```
> Options can then be
> 
> ```
> {
>     timeout: 0,
>     headers: {
>         'User-Agent': 'sdfsddddssfs',
>         'App-Version': 'this new app version i didnt plan when i first built this api'
>     }
>     ...whatever else user needs
> }
> ```
> Plus i dont care about your documentation if it's only for constructor

Updated!

File: lib/bird.js [hunk 1]
```diff
@@ -1,384 +1,385 @@
...
-          Authorization: `Bird ${this.accessToken}`
-        });
-        return axiosConfig;
-      },
-      error => Promise.reject(error)
-    );
-  }
 
-  setAccessToken(accessToken) {
-    this.accessToken = accessToken;
```
Comment (jzarca01) [inline lib/bird.js]: You didn't use the last version

File: lib/bird.js [hunk 2]
```diff
@@ -1,384 +1,385 @@
...
-      headers: {
-        'Device-id': faker.random.uuid(),
-        Platform: 'ios',
-        'App-Version': '3.0.5'
-      }
-    });
 
-    this.accessToken = null;
+    constructor(options = { }) {
+        var params = Object.assign({
```
Comment (jzarca01) [inline lib/bird.js]: Use ES6's spread operator, just like in my comments

File: lib/bird.js [hunk 3]
```diff
@@ -1,384 +1,385 @@
-const axios = require('axios');
-const faker = require('faker');
-const moment = require('moment');
```
Comment (jzarca01) [inline lib/bird.js]: Why are the ; gone ?

**Comment by jzarca01** [comment]
Closed for inactivity

**Pull request closed without merging**